src/g_game.c

```diff
@@ -3485,6 +3485,8 @@ void G_BeginRecording(void)
     for (i=0; i<4; i++)  // intentionally hard-coded 4 -- killough
       *demo_p++ = playeringame[i];
   }
+
+  doomprintf("Demo Recording: Version %d, Skill %d", demo_version, gameskill + 1);
```

Comment:
I think only port developers know about `demo_version`. Maybe we can print complevel  or filename.

Reply:
Yes, right. Do we already have a demo_version to complevel string converter?

Reply:
No, only complevel for demo_version. I think the demo file name is more useful. You can even tell the attempt number from it.

Reply:
That's what DSDA-Doom print:
![image](https://user-images.githubusercontent.com/5077629/178539887-b56a278d-db8c-4c5d-952d-716b8e8184d6.png)